src/functions_framework/_http/gunicorn.py

```diff
@@ -38,3 +64,13 @@ def load_config(self):
 
     def load(self):
         return self.app
+
+
+class GThreadWorkerWithTimeoutSupport(ThreadWorker):  # pragma: no cover
+    def handle_request(self, req, conn):
+        with ThreadingTimeout(TIMEOUT_SECONDS):
+            super(GThreadWorkerWithTimeoutSupport, self).handle_request(req, conn)
+
+
+def _is_truthy(s):  # pragma: no cover
+    return str(s).lower() in ("yes", "y", "1", "yeah", "true", "t")
```

Comment:
nit: is this necessary? It seems like we'd be fine to just accept "true" if the env var / feature is documented

Reply:
sure, let's just yank it